src/main/charts/confluence/values.yaml

```diff
@@ -183,6 +183,18 @@ synchrony:
     periodSeconds: 1
     # -- The number of consecutive failures of the Synchrony container readiness probe before the pod fails readiness checks
     failureThreshold: 30
+  resources:
+    jvm:
+      # -- The maximum amount of heap memory that will be used by the Synchrony JVM
+      maxHeap: "1g"
+      # -- The minimum amount of heap memory that will be used by the Synchrony JVM
+      minHeap: "1g"
+      # -- The memory allocated for the Synchrony stack
+      stackSize: "2048k"
+    container: 
+      requests:
+        cpu: "2"
```

Comment:
We've included the JVM arg `XX:ActiveProcessorCount` in the other product `values.yaml`  see https://github.com/atlassian-labs/data-center-helm-charts/blob/master/src/main/charts/bitbucket/values.yaml#L131

It should probably be set of synchrony too.

Reply:
Ahh this might not be necessary actually because its being set in `synchrony-start-script.yaml `

Reply:
Yeah, it gets set in the generated script, so isn't required here.

Personally I'm not a fan of putting the JVM flags in `values.yml`. IMHO it should hold facts about the infrastructure; deriving settings from those values is the job of the template. Separation of code from data, basically. But I'm not sure what is idiomatic in Helm though.

Reply:
@tarka fair point. If we decide to use [helmfile](https://hello.atlassian.net/wiki/spaces/ITPLAT/pages/1170780454/k8s+-+Managing+our+stacks+with+helmfile) it will help on this front.